### PR TITLE
[LOOP-4652] Replace AnyView

### PR DESCRIPTION
**Pull request by Camji55**
https://tidepool.atlassian.net/browse/LOOP-4652

### CODE REVIEW

**Review by nhamming — APPROVED** [review]
LGTM

File: LoopKitUI/View Controllers/DismissibleHostingController.swift [hunk 1]
```diff
@@ -8,22 +8,56 @@
 
 import SwiftUI
 
-public class DismissibleHostingController: UIHostingController<AnyView> {
+public struct _DismissibleHostingView<Content: View>: View {
```
Comment (ps2) [inline LoopKitUI/View Controllers/DismissibleHostingController.swift:11]: What is the underscore communicating here? It's not private, as the type is marked public.

**Review by ps2 — COMMENTED** [review]
Looks good! The removal of wrappers in those switch statements are nice! Just a question about naming.

File: LoopKitUI/View Controllers/DismissibleHostingController.swift (hunk 1 above)
Comment (Camji55) [reply LoopKitUI/View Controllers/DismissibleHostingController.swift:11]: This follows a convention used by Apple for code that must be declared public (it's used in the type declaration of a public function) but should not be used without caution. The DismissibleHostingController initializer will wrap it's content in this view and it should never be used directly.

**Pull request merged**